### DISCUSSION TITLE
FeedDocumentEncryptionDetails

### DISCUSSION THREAD

**Discussion by JoshLucio** [amazon-php/sp-api-sdk#198] [Q&A] [answered]
I'm trying to locate this in the code. https://developer-docs.amazon.com/sp-api/docs/feeds-api-v2020-09-04-reference#feeddocumentencryptiondetails

The encryption portion appears to be missing.

**Comment by norberttech** [comment, marked as answer]
Hey, you are reffering to 2020-09-04 version, there is newer version of this API  2021-06-30, I believe they removed endpoint you are looking for.

**Reply by JoshLucio** [reply]
Thank you, you are correct, looks like they removed the encryption.
